### DISCUSSION TITLE
View Mode for android app

### DISCUSSION THREAD

**Discussion by saswatasarkar13** [rustdesk/rustdesk#8689] [General]
I really like the view mode in rustdesk desktop app. But sometimes I want to see a screen on rustdesk android client without accidentally touching or controlling the screen. So want to have a view mode in rustdesk android client also.

Thanks for reading this.

**Comment by rustdesk** [comment]
@21pages

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/8707
